planner/cascades/testdata/transformation_rules_suite_out.json

```diff
@@ -576,6 +576,52 @@
           "    TableScan_24 table:t"
         ]
       },
+      {
+        "SQL": "(select a from t) union all (select b from t) limit 2;",
+        "Result": [
+          "Group#0 Schema:[Column#25]", 
+          "    Limit_8 input:[Group#1], offset:0, count:2", 
+          "Group#1 Schema:[Column#25]", 
+          "    Union_5 input:[Group#2,Group#3]", "Group#2 Schema:[Column#25]", 
```

Comment:
The output format here is not friendly.  `Group#2` should be in a new line.

Reply:
Good catch, will fix.